src/request/sign-transaction/SignTransaction.js

```diff
@@ -178,14 +216,14 @@ class BaseLayout {
     }
 
     run() {
-        // Go to start page
-        window.location.hash = BaseLayout.Pages.CONFIRM_TRANSACTION;
-
         // Async pre-load the crypto worker to reduce wait time at first decrypt attempt
         Nimiq.CryptoWorker.getInstanceAsync();
```

Comment:
Unrelated to this PR: Is it possible to only load the CryptoWorker when the key is stored encrypted? Do we know this detail here already? The info should be contained in the `parsedRequest.keyInfo.encrypted`, right?